src/wp-includes/capabilities.php

```diff
@@ -1088,10 +1088,9 @@ function is_super_admin( $user_id = false ) {
 		if ( is_array( $super_admins ) && in_array( $user->user_login, $super_admins, true ) ) {
 			return true;
 		}
-	} else {
-		if ( $user->has_cap( 'delete_users' ) ) {
-			return true;
-		}
+	} elseif ( $user->has_cap( 'delete_users' ) ) {
+
```

Comment:
There is an unneeded empty line here

Reply:
@audrasjb Removed empty line with latest commit.